src/sql/workbench/api/browser/mainThreadDataProtocol.ts

```diff
@@ -553,7 +553,8 @@ export class MainThreadDataProtocol extends Disposable implements MainThreadData
 
 	public $registerExecutionPlanProvider(providerId: string, handle: number): void {
 		this._executionPlanService.registerProvider(providerId, <azdata.executionPlan.ExecutionPlanProvider>{
-			getExecutionPlan: (planFile: azdata.executionPlan.ExecutionPlanGraphInfo) => this._proxy.$getExecutionPlan(handle, planFile)
+			getExecutionPlan: (planFile: azdata.executionPlan.ExecutionPlanGraphInfo) => this._proxy.$getExecutionPlan(handle, planFile),
```

Comment:
What's the meaning behind using a "$"? Is that how we distinguish our API method calls from regular method calls?